translate/src/modules/user/components/UserNotification.tsx

```diff
@@ -62,6 +84,8 @@ const Comment = ({
         }
       </Linkify>
     </div>
+
+    <DateDisplay date={date} date_iso={date_iso} />
```

Comment:
This position of `DateDisplay` shoud remain where the position of `ReactTimeAgo` was, otherwise the timestamp could appear very low on notifications with custom messages for example.